Content.Server/Spreader/KudzuComponent.cs

```diff
@@ -11,4 +13,26 @@ public sealed class KudzuComponent : Component
     /// </summary>
     [DataField("spreadChance")]
     public float SpreadChance = 1f;
+
+    /// <summary>
+    /// How much damage is required to reduce growth level
+    /// </summary>
+    [DataField("growthHealth")]
+    public float GrowthHealth = 10.0f;
+
+    /// <summary>
+    /// How much damage is required to prevent growth
+    /// </summary>
+    [DataField("growthBlock")]
+    public float GrowthBlock = 20.0f;
+
+    /// <summary>
+    /// How much the kudzu heals each tick
+    /// </summary>
+    [DataField("damageRecovery")]
+    public DamageSpecifier DamageRecovery = default!;
```

Comment:
If this is required then specify required in the datafield as right now it will just cause a nullref and the linter won't catch it.

Reply:
I've made it properly optional, and tested when it isn't specified locally.